lib/publiccloud/utils.pm

```diff
@@ -24,12 +24,15 @@ use version_utils;
 our @EXPORT = qw(select_host_console is_publiccloud is_byos is_ondemand is_ec2 is_azure is_gce);
 
 # Select console on the test host, regardless of the TUNNELED variable.
-sub select_host_console() {
+sub select_host_console {
+    my (@args) = @_;
     if (check_var('TUNNELED', '1')) {
-        select_console('tunnel-console');
+        select_console('tunnel-console', @args);
     } else {
-        select_console('root-console');
+        select_console('root-console', @args);
     }
+    send_key "ctrl-c";
```

Comment:
Isn't this only needed for the `tunnel-console` ?

Reply:
not sure. but it doesnt look harmful anyway

Reply:
Hmmmm. In theory this should be side-effect free. On the other hand it alters the program flow.

Yet, I tend to agree. This should be safe to do on both, the `root-console` and the `tunnel-console`, as CTRL+C would get redirected though `ssh`. Should be safe.

What does @pdostal thinks about this?

Reply:
I'd keep it and we will see in the future